config/routes.rb

```diff
@@ -15,6 +15,8 @@
     root 'case_studies#index', as: 'dashboard'
 
     get 'data-layers/:id', to: 'data_layers#index'
```

Comment:
DataLayer is a resource in Rails, so if we are adding multiple routes for this resource is best practice to use the default rails helpers, by replacing the two existing lines with:

`resources :data_layers, only: [:destroy, :show], as: "data-layers"`

this should cover the two routes that exist already the get (which is a show action in the controller) and the delete (which is the destroy action). the `as` at the end makes sure that the routes that were using `data-layers` (with the dash instead of underscore) will be used.

You can compare the results of the two methods with: `rake routes | grep data_layers` in your console.